### PR TITLE
Add ADConnectDump

### PR DESCRIPTION
**Pull request by nos3curity**
This toolkit offers several ways to extract and decrypt stored Azure AD and Active Directory credentials from Azure AD Connect servers.

This PR is blocked until @269 is resolved.

### CODE REVIEW

File: packages/adconnectdump.vm/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'ADConnectDump'
+$category = 'Cloud'
```
Comment (Ana06) [inline packages/adconnectdump.vm/tools/chocolateyinstall.ps1:5]: As in https://github.com/mandiant/VM-Packages/pull/253, this is another category we don't have at the moment (see https://github.com/mandiant/VM-Packages/issues/38). We need a linter and further discussion on this.

**Review by Ana06 — COMMENTED** [review]
This looks great, thank @nos3curity. We need some discussion about the categories before merging this. We can add new categories, but the problem is we can't change a package of category (as this break for example updates) so it needs to be a meditated/agreed decision. Let's discuss this in #269

**Comment by day1player** [comment]
We need to discuss how we are going to handle Python tools. Since ADConnectDump is a python tool, should we have Python as a dependency?

**Comment by mr-tz** [comment]
Yes, I'd add the needed Python version as dependency.

**Comment by nos3curity** [comment]
Added dependency. Should be ready to merge.

File: packages/adconnectdump.vm/adconnectdump.vm.nuspec [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>adconnectdump.vm</id>
+    <version>0.0.0.20230320</version>
+    <authors>fox-it</authors>
+    <description>This toolkit offers several ways to extract and decrypt stored Azure AD and Active Directory credentials from Azure AD Connect servers.</description>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="python2" />
```
Comment (mr-tz) [inline packages/adconnectdump.vm/adconnectdump.vm.nuspec:10]: Does this work with Python3? We're thinking about removing Py2 (see #301).

Comment (day1player) [reply packages/adconnectdump.vm/adconnectdump.vm.nuspec:10]: It does not! :(

Comment (Ana06) [reply packages/adconnectdump.vm/adconnectdump.vm.nuspec:10]: I think this is ok for tools that don't work on python 3.

**Review by Ana06 — APPROVED** [review]

File: packages/adconnectdump.vm/adconnectdump.vm.nuspec (hunk 2 above)
Comment (Ana06) [reply packages/adconnectdump.vm/adconnectdump.vm.nuspec:10]: I have just realised that there is an issue with this as it mess up the path: https://github.com/mandiant/VM-Packages/issues/337

**Pull request merged**